code/src/index.css

```diff
@@ -1,13 +1,120 @@
+/* GENERAL STYLING FOR ENTIRE WEBPAGE */
 body {
+  box-sizing: border-box;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-family: Monospace;
+  background-color: #FFF9F9;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+/* STYLING OF HEADER AND ITS CONTENT */
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin: 10vw;
+  padding: 5vw;
+  background: #EEEEEE;
+  border: 1px solid black;
+  box-shadow: 5px 5px;
+}
+
+/* STYLING OF FEED AND ITS CONTENT */
+.thoughts-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.thought-box {
+  width: 70%;
+  margin-bottom: 20vw;
+  padding: 10vw;
+  background: white;
+  border: 1px solid black;
+  box-shadow: 5px 5px;
+}
+
+/* STYLING OF BUTTONS */
+.form-button {
+  width: fit-content; 
+  margin-top: 10px;
+  padding: 10px;
+  border: none;
+  border-radius: 10px;
+  background: #ECA6A6;
+  font-family: Monospace;
+}
+
+.like-button {
+  border: none;
+  margin-top: 2px;
+  border-radius: 50%;
+  background: #ECA6A6
+}
+
+/* STYLING OF SMALLER ELEMENTS AND FONTS */
+p, button, textarea {
+  font-size: large;
+}
+
+.date {
+  font-style: italic;
+}
+
+/* MEDIA QUERIES */
+@media (min-width: 667px) {
+  form {
+    width: 50%;
+    margin: 5vw;
+    padding: 5vw;
+  }
+
+  .thought-box {
+    width: 50%;
+    margin: 5vw;
+    padding: 5vw;
+  }
+
+  p, button, textarea {
+    font-size: larger;
+  }
+}
+
+@media (min-width: 1024px) {
+  form {
+    width: 40%;
+    margin: 1vw;
+    padding: 1vw;
+  }
+
+  .thought-box {
+    width: 40%;
+    margin: 1vw;
+    padding: 1vw;
+  }
+
+  p, button, textarea {
+    font-size: larger;
+  }
```

Comment:
My intention here was to make the font size larger here than under a small breakpoint: first the font style is large and then larger. But perhaps this wont work either?

Reply:
I am not 100% sure about this as I myself have never used larger for font sizes. When I inspect your code in the browser it shows the mobile views as having a font size of 16px which is the default font size if you don't specify one. Then where you have a media query for 667px the font size goes down to 15.6px. And I only noted this because both your 667px and 1024px media query still have a font size of 15.6px. So they are the same.

I am not sure what the larger does because when I inspected it the font size in px actually seems smaller, from 16px to 15.6px. I don't know if larger is working on your p, button, and text area elements.